db/migrate/20171121082256_update_centos_installation_media.rb

```diff
@@ -0,0 +1,13 @@
+class UpdateCentosInstallationMedia < ActiveRecord::Migration
+  def change
+    centos_media = Medium.unscoped.where(
+      :name => 'CentOS mirror',
+      :os_family => 'Redhat',
+      :path => 'http://mirror.centos.org/centos/$version/os/$arch'
+    )
```

Comment:
you can use `.update_all(:path, 'http://mirror.centos.org/centos/$major/os/$arch')` to update them all in one shot without iteration.

Reply:
unless you want to add a condition to only update media with unchanged path, that might be useful for users who changed it

Reply:
we already select only those that have the old path in the where clause so if the user changed it it wouldn't update

Reply:
right